sysutils/rawhide/Portfile

```
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+PortGroup           legacysupport 1.0
+
+name                rawhide
+version             3.2
+
+description         (rh) find files using pretty C expressions
+long_description    (rh) find files using pretty C expressions
+homepage            https://raf.org/rawhide/
+license             GPL-3+
+platforms           darwin
+categories          sysutils
+depends_lib         port:pcre2 port:libmagic
+maintainers         raf.org:raf openmaintainer
```

Can you also add your Github handle?  If a PR is opened against this port, Github handles are tagged in and receive notification.

```suggestion
maintainers         {raf.org:raf @macportsraf} openmaintainer
```

---

Thanks. Done.